commons/monitor/questionnaire/questionnaire_application_usage_log.avsc

```diff
@@ -0,0 +1,16 @@
+{
+  "namespace": "org.radarcns.monitor.questionnaire",
+  "type": "record",
+  "name": "QuestionnaireApplicationUsageLog",
+  "doc": "Schema of reporting events when opening the app, clicking notifications, and starting and finish questionnaires.",
+  "fields": [
+    { "name": "time", "type": "double", "doc": "Device timestamp in UTC (s)." },
+    { "name": "eventType", "type": {
+      "name": "UsageEventType",
+      "type": "enum",
+      "doc": "Android activity usage event type.",
+      "symbols": ["NOTIFICATION_OPEN", "APP_OPEN", "NOTIFICATION_FOREGROUND", "QUESTIONNAIRE_STARTED", "QUESTIONNAIRE_FINISHED", "QUESTIONNAIRE_CLOSED", "OTHER", "UNKNOWN"]
```

Comment:
Could you please document each of the events? Because now their meaning is a bit ambiguous. Does finished mean completed? Does closed mean cancelled and/or completed?